src/reducers/index.js

```diff
@@ -1,7 +1,7 @@
-import undoable, { excludeAction } from 'redux-undo'
+import undoable, { excludeAction, groupByActionTypes } from 'redux-undo'
```

Comment:
groupByActionTypes is not being used anywhere

Reply:
Removed